src/routes/lexicon/+page.svelte

```diff
@@ -35,20 +32,27 @@
 
     let selectedLetter = alphabets.vernacular[0];
     let selectedWord = null;
+    $: showBackButton = selectedWord ? true : false;
     let defaultReversalKey = Object.keys(reversalAlphabets[0])[0];
     let loadedReversalLetters = new Set();
     let reversalWordsList;
     let vernacularWordsList;
-    let selectedLanguage = vernacularLanguage;
+    let vernacularLanguage;
     let scrollContainer;
+    let wordIds;
+    let selectedLanguage;
 
     // Subscribe to stores
     currentReversalLettersStore.subscribe((value) => (loadedReversalLetters = new Set(value)));
```

Comment:
I didn't notice this code from the previous PRs.  This is not the correct way to use stores.